workspaces/adventure-pack/goodies/python3/src/traverse_level_order/test.py

```diff
@@ -0,0 +1,82 @@
+from . import *
+
+
+def test_empty_tree() -> None:
+    assert list(traverse_level_order(None)) == []
+
+
+def test_root_only() -> None:
+    root = TreeNode(10)
+    assert [
+        [node.val for node in nodes] for nodes in traverse_level_order(root)
```

Comment:
It might be nice to write a helper function for this, I believe if it's not prefixed with `test_` it won't be run as a test, and we can make it a bit easier to write the tests.

Reply:
I was considering returning a generator as well for the helper function, but opted for a list for simplicity.